climada/entity/exposures/base.py

```diff
@@ -906,15 +906,29 @@ def copy(self, deep=True):
             **metadata
         )
 
-    def write_raster(self, file_name, value_name='value', scheduler=None):
+    def write_raster(self, file_name, value_name=['value'], scheduler=None):
         """Write value data into raster file with GeoTiff format
 
         Parameters
         ----------
         file_name : str
-            name output file in tif format
+            output tif file to write to
+        value_name : str or list of str
```

Comment:
Should `value_name` potentially be replaced by a plural? E.g. `values`

Reply:
That would break backward compatibility, not that anyone seems to use this. I'm happy to leave it as it is.